README.md

```diff
@@ -15,7 +15,11 @@ On [Packagist](https://packagist.org/packages/data-values/data-values):
 
 ## Requirements
 
-* PHP 5.3 or later
+These PHP versions are supported:
+
+* PHP ~5.3
```

Comment:
I don't think the `~` character gives enough information. A common way to say "PHP 5.3 or later" is "PHP 5.3+". I have never seen the `~` outside of `composer.json` files and still find it confusing every time I see it. Please be bold.